scripts/ci/sandboxed_web_e2e.py

```
@@ -221,7 +221,8 @@ def _probe_isolation_capability(backend: str) -> None:
             "--dev",
             "/dev",
             "--tmpfs",
-            "/tmp",
+            # This is the isolated namespace's tmpfs target, not a host temp path.
+            "/tmp",  # nosec B108
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)